packages/libs/src/sdk/api/tracks/types.ts

```diff
@@ -135,6 +136,14 @@ export const createUploadTrackMetadataSchema = () =>
         })
         .strict()
     ),
+    stemOf: z.optional(
+      z.object({
+        category: z.enum(
```

Comment:
Is category required? I know on the client we default to `OTHER`

Reply:
yeah good point- can make that change!